components/google-cloud/google_cloud_pipeline_components/experimental/HyperparameterTuningJob/component.py

```diff
@@ -0,0 +1,213 @@
+import json
```

Comment:
follow other components and name the directory hyperparameter_tuning_job instead of HyperparameterTuningJob